azurePipeline.yml

```diff
@@ -1,3 +1,11 @@
+trigger:
+  branches:
+    include:
+      - '*'
+  tags:
+    include:
+      - v*
```

Comment:
Why do we need a tag starting with v? Is this pipeline only for release?

Reply:
the release pipeline is triggered by the build pipeline. In the past, we didn't execute the build pipeline when a tag is set, thus, the release pipeline was never automatically triggered.